### DISCUSSION TITLE
disappeared iota

### DISCUSSION THREAD

**Discussion by davidlnch** [iotaledger/firefly#8015] [Help & Support]
My iotas have disappeared from my firefly wallet. All of them I had and also a recent transaction. I don't know how to claim or recover them. Thanks for your help

**Comment by begonaalvarezd** [comment]
If you recently updated your wallet, you might want to run the balance finder, go to settings > advanced > balance finder

**Reply by begonaalvarezd** [reply]
search until all your funds appear

**Reply by davidlnch** [reply]
i did it with no results. A few days ago I received a suspicious transaction to claim a reward in iota token and I think that when I opened a link my wallet was hacked. In the transaction finder, a transaction appears from all my iota to an address but I did not make any transaction, and it does not appear reflected in my firefly wallet either. A day later I made a transfer from Binance to my Firefly wallet. the transaction is recorded, but the iotas are not in my balance. I also included some iota to increase the voting weight in governance. These iota, tooMy iotas have disappeared from my firefly wallet. All of them I had and also a recent transaction. I don't know how to claim or recover them. they have disappeared. Today I made a new transaction from Binance and this time my iota are in the wallet balance.

**Reply by begonaalvarezd** [reply]
Firefly doesnt show all transactions that happened outside firefly because the nodes in IOTA do pruning so transactions can be missing in the wallet, for an accurate transaction history you should check all your addresses (next to your balance in firefly, there is a ".." menu -> balance history) and search them in the explorer https://explorer.iota.org/mainnet

If what you are talking about was a supposed "iota lottery" that asked to go to a website that looked like firefly but web, that was a scam, so you might have been hacked. You should never enter your mnemonic or private keys in unverified/untrusted wallets. https://twitter.com/iota/status/1753444293979603187

If you got hacked, I suggest you stop using that mnemonic and create a new one

**Reply by davidlnch** [reply]
thank so much for your help
